dev/conductor/core/test/start_test.dart

```diff
@@ -884,13 +935,27 @@ void main() {
       expect(state.releaseType, ReleaseType.STABLE_INITIAL);
     });
 
-    test('StartContext gets engine and framework checkout directories after run', () async {
+    test('mirrors are interpreted from github username', () async {
```

Comment:
I don't think you need this test--it's not actually testing any code under lib/src.

Reply:
If you want to test the interpolation, you can hard-code the `engineMirror` and `frameworkMirror` strings on lines 26 & 28, and the `FakeCommand`'s will verify that the conductor is interpolating them correctly.

Reply:
Yeah, this is the test I was using to figure out what was going on, it can be removed.